<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditCommonEvents</class>
 <widget class="QWidget" name="EditCommonEvents">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Trigger:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Condition:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="line_name"/>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="combo_trigger">
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Autorun</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Parallel</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="SwitchVariableWidget" name="switch_widget" native="true">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QListView" name="list"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SwitchVariableWidget</class>
   <extends>QWidget</extends>
   <header>dialogs/switchvariablewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>line_name</tabstop>
  <tabstop>combo_trigger</tabstop>
  <tabstop>list</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
